tbd/template/template.go

```diff
@@ -0,0 +1,64 @@
+package template
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"path/filepath"
+)
+
+type MissingError string
+
+func (e MissingError) Error() string {
+	return fmt.Sprintf("no template exists with name '%s'", string(e))
+}
+
+type Templates map[string]*template.Template
+
+type Template func(io.Writer, interface{}) error
+
+// Get name template from the Templates collection.
+func (t Templates) Get(name string) Template {
```

Comment:
This could return an error if `t[name] == nil`, but given how we use it I think that would make the code really annoying to read. In reality if you manage to get a `MissingError` in production you've clearly not tested that your change works.